### ISSUE TITLE
Effekte ohne Text

### ISSUE THREAD

**Issue by AdamK-DE** [klein0r/ioBroker.awtrix-light#121]
### I'm sure that

- [X] This issue is still present in the **current beta version** of this adapter
- [X] There is no other (open) issue with the same topic (use the search!)
- [X] This issue is not described in the adapter documentation / FAQ (read the docs!)

### Version of awtrix-light firmware

x

### The problem

Effekte ohne Text gehen nicht.. Man kann tricksen und bei den Augen schwarze font-farbe wählen und ein "." setzen das man dann gar nicht sieht aber wieso geht das nicht ganz ohne was? ideal wäre es wenn man die effekte direkt per DP starten könnte :) so bastle ich mir dann ne fake app dafür...

ps: wieso ists so ein krampf hier was posten zu können. pull request bekomme ich gar nicht hin......... 
klar das unnötiges/zuviel nervt aber so krampfig dann?! :(

### Version of nodejs

03232

### Version of ioBroker js-controller

32323

### Version of adapter

32323

**Comment by klein0r** [comment]
Das sollte seit Version 0.9.0 gehen.

https://github.com/klein0r/ioBroker.awtrix-light/blob/ab63051a99710af5a58a5d8f543d65407c20ab8d/CHANGELOG_OLD.md?plain=1#L42-L49

> wieso ists so ein krampf hier was posten zu können

Warum lässt Du denn die Hälfte der Infos weg? So kann niemand helfen...

**Comment by AdamK-DE** [comment]
hier: app mit text DP angelegt.
DP mit "" befüllt: nichts.

evtl geht's nur wenn die app ohne Text DP ist? Das testete uch noch nicht.

**Comment by klein0r** [comment]
Es fehlt nach wie vor ein Log und die nötigen Informationen.
Ich kann Dir so nicht helfen, weil es nur Raterei ist. Ich helfe gerne bei wirklichen Issues - nur muss ich die irgendwie nachvollziehen oder reproduzieren können. Ich habe allerdings keine Ahnung welche Versionen Du so verwendest oder was genau an Dein Gerät gesendet wird. Bitte sieh mir nach, dass ich für diese Raterei keine Zeit aufbringen kann. Daher ist das Issue-Template hier ja auch so definiert - damit ich nicht alles erfragen muss.

**Comment by klein0r** [comment]
> das man später noch icon oder textfarbe verändern könnte

Gibt es doch mit den Experten-Apps...

Aber: Das hier hat ja alles nichts mit dem Issue zu tun, dass Du keine Apps ohne Text (nur mit Effekt) anlegen kannst. Daher Offtopic. Du wechselst nach wie vor im Minutentakt das Thema.

**Comment by AdamK-DE** [comment]
..du brauchst ne anständige doku site! ;) nein das github ding ist keine...
schau mir das expert zeugs gleich an..

und ja, weil man dich ja sonst kaum erreichen kann und das hier so ein krampf ist.. github mag für programmierer toll sein aber für user die nem wunsch/bug/sonstwas schreiben wollen ists ein absoluer krampf.. 99% rennen garantiert davon wenn sie diese seite hier sehen ;) vorallem wenn man erstmal 30 "werte" eintippen muss um dann irgendwas schreiben zu dürfen beidem die aber eh absolut irrelevant sind... garantiert scheitert da schon ne vielzahl der user daran die versionen überhaupt heraus zu bekommen.........

**Comment by klein0r** [comment]
Ich halte mich da an den Standard wie ioBroker-Adapter dokumentiert werden sollen.

Warum kann man mich sonst nicht erreichen? Ich bin sehr aktiv im Forum, auf YouTube, auf Instagram und hier auf GitHub. Wann hast Du mal länger als 1 Tag auf eine Antwort gewartet?

Und GitHub ist für Issue-Tracking. Nicht für allgemeine Fragen. Dafür gibt es das Forum. Den Link zum Thread hast Du ja beim anlegen des Issues gesehen... Und natürlich muss ich alles abfragen. Wie soll ich sonst helfen? Hier schlagen ständig Leute mit uralten Versionen auf und dann kann ich direkt sagen "das ist der Grund".

Du kannst ja mal ein paar hundert Kommentare im Monat hier schreiben und Issues beantworten. Wenn 95% der Fragen dann sind "und welche Versionen hast Du im Einsatz" macht so ein Template das Leben deutlich einfacher. Ggf. die andere Seite gibt sich auch etwas Mühe mit Fehler-Reports und versucht so viel wie möglich zu liefern. Daran scheitert es meistens...

**Comment by AdamK-DE** [comment]
ja, verstehe dich da schon..  aber hatte es leider schon sooo oft das man sich bei neuen adaptern einen abquelt und später merkt das es ja viel einfach gegangen wäre.. davon sah man aber anfangs niergens was.. ich schaute deine videos dazu und überflog die sites.. das expert ding sah ich nicht. könnte nun direkt schonwieder darauf umstellen.. argh! ;) aber klar, kam sicher erst später dazu..

egal mach zu hier.

ps: wooooo sind die features die HA dazu noch mitbringt?
- icon das mit dem text wegscrollt (=echt brauchbar)
- 32x8 icons/anims (gif 2 json nötig?)


ps: kontakt per FB war schön einfach. echte foren nutzt 2024 doch kaum noch jemand ;) und youtube ist eher für kurz mal was..

**Comment by AdamK-DE** [comment]
https://blueforcer.github.io/awtrix-light/#/icons

ist verschwunden. ist auch bei dir bei den apps->icon verlinkt. 404

**Comment by AdamK-DE** [comment]
und: evtl kannst du das noch irgendwie fixen. mein DP ändert sich jede sec 1A, blockly aktuallisiert aus alexa timer.. aber gesendet wird nur alle 3-4 sec..  seltsam.. bei text wirkt es immer nach sofort:

![awtrix(1)](https://github.com/klein0r/ioBroker.awtrix-light/assets/127183545/2c417be4-03e5-43e2-b1c4-4931fdfc41c0)

**Comment by klein0r** [comment]
> ist verschwunden. ist auch bei dir bei den apps->icon verlinkt. 404

Ist schon mit der 0.16.0 gefixt. Liegt am rebranding zu "awtrix3".

> aber gesendet wird nur alle 3-4 sec

Korrekt, um die Uhr nicht mit zig Requests zu nerven. Gab in der Vergangenheit Probleme damit, weil die Leute das alles nicht so nutzen wie gedacht... Ständig apps ein- und auszublenden für ein paar Sekunden ist ja auch nicht die Idee. Für einmalig kurz sichtbare Dinge sind Notifications gedacht.
